packages/shared/src/components/onboarding/EditTag.tsx

```diff
@@ -24,16 +28,40 @@ export const EditTag = ({
   customActionName,
   activeScreen,
 }: EditTagProps): ReactElement => {
+  const isMobile = useViewSize(ViewSize.MobileL);
   const [isPreviewVisible, setPreviewVisible] = useState(false);
   const tagsCount = feedSettings?.includeTags?.length || 0;
   const isPreviewEnabled = tagsCount >= REQUIRED_TAGS_THRESHOLD;
 
+  const [searchQuery, setSearchQuery] = useState<string>('');
```

Comment:
Just reminder we need to double check onboarding flow on testing mode.

Reply:
yep, I checked, searching works fine